src/lookup_service.cpp

```diff
@@ -150,11 +152,13 @@ LookupService::set64(const DmtcpMessage &msg)
     break;
 
   case DMTCP_KVDB_AND:
+    // If key isn't found, set the key to the given value.
     if (kvmap.find(msg.kvdb.key) == kvmap.end()) {
       kvmap[msg.kvdb.key] = msg.kvdb.value;
+      break;
```

Comment:
Here also, I assume you intend to flatten the three commits into one, which is fine.

I do have a small quibble.  I think the intention of the code will be clearer, if you reorganize this as:  `if (kvmap.find(msg.kvdb.key) == kvmap.end()) {...} else { ...}`
and then add a single `break' at the end of the if-then-else, to indicate the end of this `case` statement.

Reply:
Good point. I'll update it in a followup PR to avoid re-approval of this PR.